src/resources/js/components/Presentation.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="container">
+     <div v-if="isError" class="error">
+          <h3>{{ message }}</h3>
+     </div>
+
+     <h3>Presentation info</h3>
+     <div>
+          <h4>title : {{ presentation.title }} </h4>
+          <p>created on : {{ formatDate(presentation.date) }} </p>
+     </div>
+
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+  name: "Presentation",
+  props: ["idPresentation"], // we will send the id of the presentation through props
+  data() {
+    return {
+      presentation: {},
+      polls: [],
+      isError: false,
+      message: ''
+    };
+  },
+  beforeMount() {
+    this.getPresentationInfo();
+  },
+  methods: {
+     getPresentationInfo() {
+      //const id = idPresentation;
+      const id = 1;
```

Comment:
on the dev branch the real id should be used

Reply:
ok, I removed this line in [c8280c](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/113/commits/c8280cd09d8dea4eaa034bfaf7fff0b7357c2b69)